Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -171,6 +172,13 @@ public static void launch(FragmentActivity activity, String filePath, boolean ri
     launch(activity, new String[]{filePath}, riivolution);
   }
 
+  public static void launchSystemMenu(FragmentActivity activity)
+  {
+    Intent launcher = new Intent(activity, EmulationActivity.class);
+    launcher.putExtra(EmulationActivity.EXTRA_SYSTEM_MENU, true);
+    activity.startActivity(launcher);
+  }
```

Comment:
This is missing all the checks that are in the normal `launch`. Especially the path checks are important.